nmostesting/Config.py

```diff
@@ -55,6 +55,9 @@
 # Number of seconds to wait after performing an API action for the results to be fully visible via IS-04
 API_PROCESSING_TIMEOUT = 1
 
+# Number of seconds to wait before timing out Controller test. Set to 0 to disable timeout mechanism
```

Comment:
I did consider this.  Maybe it makes sense to use None instead of 0 - as we literally are specifying no timeout.

There is a test case that overrides the default timeout - it's the "end of tests" message, which is overridden to always have a timeout of 10 seconds.  However, given the timeout is primarily there to prevent locking in the case of automated tests, there probably isn't a compelling case to have the override - thus removing the need to have a default timeout value ( and the associated confusion in the code)